_jekyll/_layouts/default.html

```diff
@@ -12,7 +12,7 @@
         {% endif %}
     </head>
 
-    <body style="margin: 1rem;">
+    <body>

```

Comment:
Isn't this needed for the other examples? You could make a different layout for this to remove it

Reply:
Oh damn....good spot @deadlight. How do I go about removing it from the homepage? 🤔

Reply:
@deadlight updated 👍